packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/MaintenanceWindow.tsx

```diff
@@ -173,22 +171,30 @@ export const MaintenanceWindow = (props: Props) => {
     onSubmit: handleSaveMaintenanceWindow,
   });
 
+  const isLegacy = database.platform === 'rdbms-legacy';
+
+  const typographyDatabase =
```

Comment:
nitpick: I think we normally use capitalized snake case for copy constants, but someone correct me if I'm wrong!
```suggestion
  const TYPOGRAPHY_DATABASE =
```
these constants could also be moved to the top of the scope, outside the component. Alternatively, you could do something like you did with the copies in `DatabaseSettings`:

`const typographyDatabase = isLegacy ? ... : ...`

Reply:
Not sure, other places have camelCase for text copy. Personally, I'd do CAPS for const but not sure the standard.